core/consensus/src/message.rs

```diff
@@ -14,7 +14,8 @@ use protocol::traits::{
 };
 use protocol::ProtocolError;
 
-use crate::fixed_types::{FixedBlock, FixedHeight, FixedSignedTxs, PullTxsRequest};
+use crate::fixed_types::{FixedBlock, FixedHeight, FixedProof, FixedSignedTxs, PullTxsRequest};
+use core_storage::StorageError;
```

Comment:
Look at how the old code sorts `use`, you'll see some rules

Reply:
comprehended